src/utils/periods/relativePeriods.js

```diff
@@ -195,3 +240,118 @@ export const flattenedRelativePeriods = RELATIVE_PERIODS.reduce(
 
 export const isFixedPeriodType = periodType =>
     !!fixedPeriodTranslations[periodType]
+
+const formatYyyyMmDd = timestamp => {
```

Comment:
I was wondering, if moment's available here, maybe that would be easier than using the date api for all the date based calculations in this file?

Reply:
Perhaps this logic is going to be extracted from the reports-app and integrated into [dhis2/analytics ](https://github.com/dhis2/analytics). So I tried to use helpers [as present there](https://github.com/dhis2/analytics/blob/master/src/components/PeriodDimension/utils/fixedPeriods.js#L537-L546). Therefore I'd like to keep it the way it is.

Reply:
Maybe a small comment explaining that, somewhere in the beginning of this file, would be helpful. Just so people are aware that that's where the code style comes from?

Reply:
added in bec9d6e